bin/install_Forwarder.sh

```diff
@@ -0,0 +1,47 @@
+yum update -y &&
+yum install -y \
+    which-2.20 \
+    librabbitmq-0.8.0 \
+    librabbitmq-devel-0.8.0 \
+    libcurl-7.29.0 \
+    libcurl-devel-7.29.0
+
+#==============================================================================
+#           Install C++ libraries
+#==============================================================================
+mkdir -p $IIP_TMP_DIR
+
+# url, folder
+cmake_install() { 
+    cd $IIP_TMP_DIR
+    curl -L $1 -o $2.tar.gz
+    tar zxvf $2.tar.gz
+    cd $2
+    mkdir build && cd build
+    cmake ..
+    cmake --build . --target install
+}
+
+# url, folder
+make_install() { 
+    cd $IIP_TMP_DIR
+    curl -L $1 -o $2.tar.gz
+    tar zxvf $2.tar.gz
+    cd $2
+    make
+    make install
+}
+
+# boost
+cd $IIP_TMP_DIR
+curl -L https://dl.bintray.com/boostorg/release/1.66.0/source/boost_1_66_0.tar.gz -O
+tar zxvf boost_1_66_0.tar.gz
+cd boost_1_66_0
+./bootstrap.sh
+./b2 --with-test --with-log install
+
+# C++ libraries
+cmake_install "https://github.com/jbeder/yaml-cpp/archive/yaml-cpp-0.5.3.tar.gz" "yaml-cpp-yaml-cpp-0.5.3"
```

Comment:
We discussed that we need to host these tarball locally and not get them pulled directly from third party repositories. Please fetch them and host them locally on your `~/public_hml` for now.
One model I've used in the past for DES is `$URL/{path-to-resources}/{package_name}/{package_name}-{version}.tar.gz`